### ISSUE TITLE
Example project

### ISSUE THREAD

**Issue by schinckel** [funkybob/graaf#5]
Actually, it would probably be a good idea to use an example project for the test suite.

**Comment by funkybob** [comment]
Yeah, I was considering this yesterday, as I was trying to motivate myself to further documentation :)
